modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/endpoint/ServiceRegistryEndpoint.java

```diff
@@ -291,13 +301,31 @@ public Response setMaintenanceMode(@FormParam("host") String host, @FormParam("m
           @RestResponse(responseCode = SC_OK, description = "Returned the available services."),
           @RestResponse(responseCode = SC_BAD_REQUEST, description = "No service type specified, bad request.") })
   public Response getAvailableServicesAsXml(@QueryParam("serviceType") String serviceType) {
+
     if (isBlank(serviceType))
       throw new WebApplicationException(Response.status(Status.BAD_REQUEST).entity("Service type must be specified")
               .build());
+
+    Organization currentOrg = securityService.getOrganization();
+    Map<String, String> properties = currentOrg.getProperties();
```

Comment:
No strong opinion on this:

```suggestion
    Map<String, String> properties = securityService.getOrganization().getProperties();
```

Reply:
Yep.